apps/family-public-law/fpl-cron-ccd-data-migration-tool/perftest/00.yaml

```diff
@@ -6,12 +6,15 @@ spec:
   releaseName: fpl-cron-ccd-data-migration-tool
   values:
     job:
-      image: hmctspublic.azurecr.io/fpl/cron-ccd-data-migration-tool:pr-2-ade5141-20230222140930 #{"$imagepolicy":"flux-system:perftest-fpl-cron-ccd-data-migration-tool"}
+      image: hmctspublic.azurecr.io/fpl/cron-ccd-data-migration-tool:pr-101-1c7a9ab-20230308102945 #{"$imagepolicy": "flux-system:perftest-fpl-cron-ccd-data-migration-tool"}
```

Comment:
should that whitespace be there?

Reply:
This is similar to the main FLP service - https://github.com/hmcts/cnp-flux-config/blob/8180108b07a23ad6ffa342fa6cce7ded41930c6f/apps/family-public-law/fpl-case-service/demo.yaml#L10. So, I don't there should be any issue.